lib/editor/actions/editor.js

```diff
@@ -167,56 +167,32 @@ export function lockEditorFeedSource (feedId: string) {
  * Stop job time function stored with timer ID stored in state.
  */
 function stopCurrentTimer (state: AppState) {
+  // FIXME
   const {timer} = state.editor.data.lock
   if (timer) clearInterval(timer)
 }
 
 /**
- * Stops the lock timer.
- */
-export function stopLockTimer () {
-  return function (dispatch: dispatchFn, getState: getStateFn) {
-    stopCurrentTimer(getState())
-  }
-}
-
-/**
- * HTTP call to the feed lock endpoint.
+ * HTTP call to .
```

Comment:
Is this what this comment was before #815?

Reply:
Yes it is a straight revert to what was there before.